www/docs/en/dev/guide/platforms/android/webview.md

```diff
@@ -29,106 +29,86 @@ components can communicate with each other, see Application Plugins.
+1. Create a project in Android Studio
 
-1. Copy the `.jar` file into the Android project's `/libs` directory.
+1. Copy the framework directory into your project's root directory and name it
+CordovaLib
+
+1. Add CordovaLib in your settings.gradle file
+
+include ':app', ':CordovaLib'
+
+1. Add that project as a library project to your main project.  This can be done in your application's build.gradle file found in
+app/build.gradle:
+
+        dependencies {
+              compile fileTree(dir: 'libs', include: ['*.jar'])
+              compile 'com.android.support:appcompat-v7:21.0.3'
+              //Add Cordova below
+              debugCompile project(path: ":CordovaLib", configuration: "debug")
+              releaseCompile project(path: ":CordovaLib", configuration: "release")
+        }
 
 1. Add the following to the application's `/res/xml/main.xml` file,
    with the `layout_height`, `layout_width` and `id` modified to suit
    the application:
 
-        <org.apache.cordova.CordovaWebView
+        <org.apache.cordova.engine.SystemWebView
             android:id="@+id/tutorialView"
             android:layout_width="match_parent"
             android:layout_height="match_parent" />
 
-1. Modify the activity so that it implements the `CordovaInterface`.
-   It should implement the included methods.  You may wish to copy
-   them from `/framework/src/org/apache/cordova/CordovaActivity.java`,
-   or else implement them on your own.  The following code fragment
-   shows a basic application that relies on the interface. Note how
-   the referenced view id matches the `id` attribute specified in the
-   XML fragment shown above:
-
-        public class CordovaViewTestActivity extends Activity implements CordovaInterface {
-            CordovaWebView cwv;
-            /* Called when the activity is first created. */
-            @Override
-            public void onCreate(Bundle savedInstanceState) {
-                super.onCreate(savedInstanceState);
-                setContentView(R.layout.main);
-                cwv = (CordovaWebView) findViewById(R.id.tutorialView);
-                Config.init(this);
-                cwv.loadUrl(Config.getStartUrl());
-            }
+1. Add the following code to setup your Activity so that you have an interface
 
-1. If the application needs to use the camera, implement the
-   following:
+        private CordovaInterfaceImpl iface = new CordovaInterfaceImpl(this);
+
+1. Add the following to intialize the WebView
+
+        //Set up the webview
+        ConfigXmlParser parser = new ConfigXmlParser();
+        parser.parse(this);
+
+        SystemWebView webView = (SystemWebView) findViewById(R.id.tutorialView);
+        webInterface = new CordovaWebViewImpl(new SystemWebViewEngine(webView));
+        webInterface.init(iface, parser.getPluginEntries(), parser.getPreferences());
+
+        webView.loadUrl(parser.getLaunchUrl());
+
+
+1. Copy the application's HTML and JavaScript files to the Android
```

Comment:
You also need to copy cordova.js from `bin/templates/project/assets/www` to that folder